packaging/installer/methods/freebsd.md

```diff
@@ -0,0 +1,17 @@
+# Install Netdata on FreeBSD
+
+You can install Netdata from either the `ports` or `packages` collections.
+
+This is how to install the latest Netdata version from source on FreeBSD:
+
+```sh
+# install required packages
+pkg install bash e2fsprogs-libuuid git curl autoconf automake pkgconf pidof Judy liblz4 libuv json-c
+
+# download Netdata
+git clone https://github.com/netdata/netdata.git --depth=100
+
+# install Netdata in /opt/netdata
+cd netdata
+./netdata-installer.sh --install /opt
+```
```

Comment:
We should probably mention somehwere here that there is an (unofficial AFAIK) Netdata package in the FreeBSD ports tree. Many users are likely to prefer that if they know about it, as ti integrates better with the rest of the system than our current installer.

Also, unless I'm mistaken, `kickstart.sh` _should_ work on FreeBSD, and that's what we should be recommending most of the time.